dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/actions/CodewindInstall.java

```diff
@@ -77,6 +81,22 @@ public static void codewindInstallerDialog() {
 		    }
 	}
 	
+	public static void codewindInstallerDialog(IProject project) {
+		Shell shell = new Shell();
+		MessageBox dialog =
+			    new MessageBox(shell, SWT.ICON_QUESTION | SWT.YES| SWT.NO);
+			dialog.setText(Messages.InstallCodewindDialogTitle);
+			dialog.setMessage("Could not add " + project.getName() + ", " + Messages.InstallCodewindDialogMessage);
```

Comment:
The InstallCodewindDialogMessage begins with a capital so this won't work.  It is better to create a new message.